### PR TITLE
systemd-cgls: add page

### PR DESCRIPTION
**Pull request by dadav**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

see  #10191

### CODE REVIEW

File: pages/linux/systemd-cgls.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# systemd-cgls
+
+> Shows the contents of the selected Linux control group hierarchy in a tree.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-cgls.html>.
+
+- To display the whole cgroup hierarchy on your system:
```
Comment (kbdharun) [inline pages/linux/systemd-cgls.md:6]:
```suggestion
- Display the whole control group hierarchy on your system:
```

File: pages/linux/systemd-cgls.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`systemd-cgls`
+
+- To see a cgroup tree of a specific resource controller (e.g. `cpu` or `memory`), execute:
+
+`systemd-cgls {{controller}}`
```
Comment (kbdharun) [inline pages/linux/systemd-cgls.md:12]:
```suggestion
- Display a control group tree of a specific resource controller:

`systemd-cgls {{cpu|memory|io}}`
```

File: pages/linux/systemd-cgls.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Show the hierarchy of a specific systemd unit:
```
Comment (kbdharun) [inline pages/linux/systemd-cgls.md:14]:
```suggestion
- Display the hierarchy of a specific systemd unit:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by acuteenvy — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**